plugin/pkg/auth/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -60,9 +62,15 @@ func NewCSV(path string) (*PasswordAuthenticator, error) {
 			info:     &user.DefaultInfo{Name: record[1], UID: record[2]},
 			password: record[0],
 		}
+		if _, exist := users[obj.info.Name]; exist {
+			isDuplicate = true
```

Comment:
Why not  just drop the warning here so you can tell them the names of all the duplicates?